content/applications/hr/payroll/contracts.rst

```diff
@@ -173,19 +182,20 @@ To add a new deduction, click :guilabel:`Add a line`. Type in a description for
 
```

Comment:
can you please elaborate on where the 'salary attachment' option/field/tab(?) can be located? I am having a hard time locating it.